spec/models/user_access_spec.rb

```diff
@@ -111,11 +143,13 @@
           it "returns the facility groups an admin can perform actions on with organization access" do
             permission_matrix.each do |admin, action, current_resource, expected_resources|
               admin.accesses.create!(resource: current_resource)
+
               expect(admin.accessible_facility_groups(action)).to match_array(expected_resources),
                 error_message(admin,
                   action,
                   expected_resources,
                   admin.accessible_facility_groups(action))
+              expected_resources.each { |resource| expect(admin.can_access?(current_resource, action)).to be_truthy }
```

Comment:
I'm buildingin onto the specs we have, because they already create a whole matrix of data that is very useful to assert on (and I don't want to create any more objects than we already are here for testing...).